ports/unix/Makefile

```diff
@@ -44,10 +44,18 @@ CFLAGS += $(INC) $(CWARN) -std=gnu99 -DUNIX $(CFLAGS_MOD) $(COPT) -I$(VARIANT_DI
 
 # Debugging/Optimization
 ifdef DEBUG
+COPT_LEVEL ?= -O0
+COPT_NDEBUG ?=
 CFLAGS += -g
-COPT = -O0
 else
-COPT = -Os -fdata-sections -ffunction-sections -DNDEBUG
+COPT_LEVEL ?= -Os
+COPT_NDEBUG ?= -DNDEBUG
+COPT_EXTRA += -fdata-sections -ffunction-sections
+endif
+
+COPT += $(COPT_LEVEL) $(COPT_EXTRA) $(COPT_NDEBUG)
```

Comment:
Perhaps COPT_EXTRA should be at the end of this line so it can override previous things?

Reply:
But, wouldn't it be simpler to just keep it as it was (with just COPT) and then enabled debugging (`DEBUG = 1`) for the coverage variant?

Reply:
OK, I've updated as suggested.